code/modules/clothing/suits/jobs.dm

```diff
@@ -165,6 +165,24 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO
 	price_tag = 50
 
+//Chief Engineer/Technomancer Exultant
+/obj/item/clothing/suit/storage/te_coat
+	name = "exultant coat"
+	desc = "A sturdy and proud crimson coat. Lightly armored, with some protection against radiation."
+	icon_state = "te_coat"
+	item_state = "te_coat"
+	blood_overlay_type = "coat"
+	body_parts_covered = UPPER_TORSO|LOWER_TORSO|ARMS
```

Comment:
Should cover legs.

Reply:
The sprite doesn't cover the legs, but aye.

Reply:
Unless this is for the heat resistance, fair nough then far as I know.

Reply:
It's long enough to do that.